third_party/2and3/jinja2/utils.pyi

```diff
@@ -1,6 +1,6 @@
 from typing import Any, Optional
 
-from markupsafe import Markup, escape, soft_unicode
+from markupsafe import Markup as Markup, escape as escape, soft_unicode as soft_unicode
```

Comment:
This is a little strange (Jinja2 exports functions from markupsafe directly), but confirmed that it's correct.